web/admin/src/Main.svelte

```diff
@@ -74,6 +75,11 @@
         <RaffleView raffleId={Number(params.raffleId)} />
       {/snippet}
     </Route>
+    <Route path="/contests/:contestId/tickets/list">
```

Comment:
Use `/contests/:contestId/tickets` path for this and move PrintableTicketsList to `/contests/:contestId/tickets/print` instead.